### DISCUSSION TITLE
After getting a new monitor, lazer loads, and immediately crashes.

### DISCUSSION THREAD

**Discussion by kevinboiiii** [ppy/osu#28560] [Q&A]
### Type

Crash to desktop

### Bug description

I had recently swapped to a new monitor, mainly one that had a higher refresh rate, 60hz to 165hz, and now lazer crashes on load.

### Screenshots or videos

https://youtu.be/JiOr0406VC8

I did not touch my keyboard or mouse after I opened osu.

### Version

2024.521.2-lazer

### Logs

Please let me know where to find the log files, because I cannot even get past the screen, let alone open settings.

**Comment by kevinboiiii** [comment]
EDIT: it seems to have mysteriously fixed itself. I don't think I did anything... here's the logs if they are still useful
[compressed-logs.zip](https://github.com/user-attachments/files/15942258/compressed-logs.zip)

**Comment by BlockHead007** [comment]
No errors in logs. What does event viewer say? Press Win+X, then on Event viewer (or just type that in search), then select applications and find .NET Runtime source with en error.
